subgraphs/morpho/src/helpers.ts

```diff
@@ -1080,3 +1187,40 @@ export function updateP2PRates(market: Market): void {
 
 export const getEventId = (hash: Bytes, logIndex: BigInt): Bytes =>
   hash.concat(Bytes.fromI32(logIndex.toI32()));
+
+export function updateProtocolTotalValueLockedUSD(
```

Comment:
Should we also extend this function to also update `totalDepositBalanceUSD` and `totalBorrowBalanceUSD`?

Reply:
Yeah, makes sense